test/example.yaml

```diff
@@ -6,8 +6,9 @@
 name: example
 templates:
   - name: "example-cluster"
-    driver: podman
-    container_runtime: cri-o
+    driver: kvm2
+    container_runtime: containerd
+    network: default
```

Comment:
This is fine, but we need a commit message that will help reviewers to accept this
change, and help future maintainers to understand this change 6 month in the future.

The title should describe what this change does, maybe:

    Switch example to kvm2 driver

The body of the commit message should explain why this change is needed, maybe:

    Podman driver is causing too much trouble lately, specially when running
    multiple clusters. Lets go back to kvm2 driver.

Reply:
resolved